TriblerGUI/widgets/trustpage.py

```diff
@@ -116,7 +116,11 @@ def plot_absolute_values(self):
 
         # Convert all dates to a datetime object
         for block in self.blocks:
-            plot_data[1].append(datetime.datetime.strptime(block["insert_time"], "%Y-%m-%d %H:%M:%S"))
+            try:
+                date = datetime.datetime.strptime(block["insert_time"], "%Y-%m-%d %H:%M:%S")
+            except ValueError:
+                date = datetime.datetime.strptime(block["insert_time"].decode("HEX"), "%Y-%m-%d %H:%M:%S")
```

Comment:
This is not something that should be fixed in the GUI: it seems that the core passes incorrect data. After a quick investigation, it seems to be caused by the `__iter__` method in a block (https://github.com/Tribler/tribler/blob/devel/Tribler/community/trustchain/block.py#L326). Simply ignoring the `insert_time` field here should work.